### ISSUE TITLE
Require image for ARM64 platform

### ISSUE THREAD

**Issue by odidev** [zulu-openjdk/zulu-openjdk#98]
I am trying to build a Zulu-OpenJDK image for alpine over ARM64 but it seems it does not have ARM64 supported tags in the [docker hub](https://hub.docker.com/r/azul/zulu-openjdk/tags?page=1&ordering=last_updated)

I am unable to build the image for arm64 since it is using Zulu binaries 

I have tried installing Zulu in the[ installation steps](https://docs.azul.com/core/zulu-openjdk/install/alpine-linux) to install Zulu in the ARM64 platform but was unable to install it by adding the apk repositories.

I have tried installation using the [tar](https://cdn.azul.com/zulu/bin/zulu16.30.19-ca-jdk16.0.1-linux_musl_aarch64.tar.gz) file but unable to find the zula16-ca binary which is used in [Dockerfile.](https://github.com/zulu-openjdk/zulu-openjdk/blob/master/alpine/16.0.1-16.30.15/Dockerfile#L11)

I would be happy to help in providing the arm64 support in Zulu-OpenJDK but it requires the arm64 supported Zulu binaries used in the Dockerfile.

Do you have any plans for releasing an ARM64 image?

May I know how the amd64 image is getting published on the Docker hub?

If interested, I will raise a PR.

**Comment by sgrinev** [comment]
NB: Zulu will provide Linux arm64 alpine images starting from zulu16.0.1:

- https://cdn.azul.com/zulu/bin/zulu16.30.19-ca-jdk16.0.1-linux_musl_aarch64.tar.gz
- https://cdn.azul.com/zulu/bin/zulu16.30.19-ca-jre16.0.1-linux_musl_aarch64.tar.gz

**Comment by davidcurrie** [comment]
I see that there are now ARM64 images for JDK 17. Any particular reason why JDK 11 images are not also made available for ARM64?

**Comment by countableSet** [comment]
+1 for arm64 JDK 11 version. 👍🏻 

Also is there a reason why the `latest` tags as well (aka like [`17-latest`](https://github.com/zulu-openjdk/zulu-openjdk/tree/master/17-latest)) don't also include an arm64 version? Can we also get a [`11-latest`](https://github.com/zulu-openjdk/zulu-openjdk/tree/master/11-latest) arm64 version?

I don't mind opening a pr for adding arm64 dockerfiles for these but not sure how likely that would be to get approved/merged.

**Comment by davidcurrie** [comment]
@countableSet - I also raised a support ticket, and the images are now available for `11-latest`.

**Comment by marcphilipp** [comment]
> arm64 images are now available for the `11` tag.

That doesn't seem to be the case for azul/zulu-openjdk-alpine, though. 🤔

**Comment by davidcurrie** [comment]
> That doesn't seem to be the case for azul/zulu-openjdk-alpine, though. 🤔

Whilst I still had the ticket open, I asked support. They say they're on their way.

**Comment by countableSet** [comment]
I wonder if I'm looking in the wrong place, but I still don't see arm64 in docker hub for these 11 tags

* ubuntu [11 tag](https://hub.docker.com/layers/zulu-openjdk/azul/zulu-openjdk/11/images/sha256-6bee572440018a7896d5f32688c68bc3ff481b1b1685354becf3fb877e391a14?context=explore)
* debian [11 tag](https://hub.docker.com/layers/zulu-openjdk-debian/azul/zulu-openjdk-debian/11/images/sha256-84866770221168216b165a2bb8dce12b7009e07dd9ea6b35f674b3b406d2bf0d?context=explore)
* alpine [11 tag](https://hub.docker.com/layers/zulu-openjdk-alpine/azul/zulu-openjdk-alpine/11/images/sha256-acf6659675579c815886fa3f1ce00f569cdf6ee5fff24207bdb88ed327591d8f?context=explore)

**Comment by davidcurrie** [comment]
No - you're looking in the right place. I was only looking at the Ubuntu image but, briefly, it was multi-arch before then reverting to AMD only. They are building the ARM images e.g. https://hub.docker.com/layers/zulu-openjdk/azul/zulu-openjdk/11.0.15-11.56.19-arm64/images/sha256-c3bd3f2fae31e356d2383f0c59bd204a8407cc6d9a83373a1eaad21c53762a22?context=explore but, for whatever reason, aren't pushing the multi-arch manifest. I still have the support ticket open but responses have slowed...

**Comment by it-mak** [comment]
Multi-arch manifest are already presented.

**Comment by koenverwimp** [comment]
The manifest is not available in azul/zulu-openjdk:17.0.10 anymore. Is this a bug or a choice?

root@BRHESDev05-Koen ~ # docker buildx imagetools inspect  azul/zulu-openjdk:17.0.10
Name:      docker.io/azul/zulu-openjdk:17.0.10
MediaType: application/vnd.docker.distribution.manifest.v2+json
Digest:    sha256:7f489993ee6e1c2020d964bb945fcd22787a0bb56b7a2f0d1721591c06a1ee7b

**Comment by theprotos** [comment]
@koenverwimp 

Thank you for making us aware of the issue. We added linux/arm64,linux/amd64 to 17.0.10

```
Name:      docker.io/azul/zulu-openjdk:17.0.10
MediaType: application/vnd.docker.distribution.manifest.list.v2+json
Digest:    sha256:e4668ec9ff7a29e780b7cdf85d38e3934a16dd600ae9d9afbcb5763396cc8ee0

Manifests:
  Name:      docker.io/azul/zulu-openjdk:17.0.10@sha256:d81cd05d5d89c66a739bb10227ad505748380001a7b3d23b97e7648a97343e7d
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/arm64/v8

  Name:      docker.io/azul/zulu-openjdk:17.0.10@sha256:12ea0d7ff3a80d924decc0cd5f3c79ebf266b7997a58ccd7dc1a8c2845ed7791
  MediaType: application/vnd.docker.distribution.manifest.v2+json
  Platform:  linux/amd64
```
